src/js/node/child_process.ts

```diff
@@ -1363,6 +1363,37 @@ class ChildProcess extends EventEmitter {
         this.emit("spawn");
       });
 
+      // Make stdio properties enumerable own properties for Object.assign() compatibility
+      // This fixes libraries like tinyspawn that use Object.assign(promise, childProcess)
+      Object.defineProperty(this, "stdin", {
```

Comment:
This must be done on the class prototype, probably in a class static initializer instead of on the constructor.